### PR TITLE
Fix wide character spacing

### PR DESCRIPTION
**Pull request by ronkok**
Fixes issue #1360.

Visual confirmation:
widespacing

### CODE REVIEW

File: test/katex-spec.js [hunk 1]
```diff
@@ -1548,6 +1548,11 @@ describe("An HTML font tree-builder", function() {
         expect(markup).toContain(span);
     });
 
+    it("should render wide characters with mord and with the correct font", function() {
+        const markup = katex.renderToString(String.fromCharCode(0xD835, 0xDC00));
+        expect(markup).toContain("<span class=\"mord mathbf\">A</span>");
+    });
```
Comment (kevinbarabash) [inline test/katex-spec.js]: We might also want a test like
```
expect("M = a + b").toBuildLike("\textbf{M = a + b}");
```
where "M = a + b" is the bolded versions.  Not sure the best way to store a string with bolded characters.

Comment (ronkok) [reply test/katex-spec.js]: I just tried something equivalent to:
```
expect("𝐀 = 𝐚").toBuildLike("\\mathbf{A} = \\mathbf{a}");
```

It turns out that they don't build in an equivalent way. `\mathbf{A}` returns a `mord` span wrapped inside a `mord mathbf` span. "𝐀" returns a single `mord mathbf` span. They render in an equivalent way, but the test doesn't like it.

**Comment by ronkok** [comment]
> This should fix the italic, script, etc. unicode chars right?

Yep. Anything with a high surrogate `\uD835`.

File: test/katex-spec.js (hunk 1 above)
Comment (edemaine) [reply test/katex-spec.js]: Can you test
```js
expect("𝐀 = 𝐚").toBuildLike("\\mathbf A = \\mathbf a");
```
?

Comment (ronkok) [reply test/katex-spec.js]: @edemaine That one works! Cool!

File: src/buildCommon.js [hunk 2]
```diff
@@ -238,7 +238,8 @@ const makeOrd = function<NODETYPE: "spacing" | "mathord" | "textord">(
     if (value.charCodeAt(0) === 0xD835) {
         // surrogate pairs get special treatment
         const [wideFontName, wideFontClass] = wideCharacterFont(value, mode);
-        return makeSymbol(value, wideFontName, mode, options, [wideFontClass]);
+        return makeSymbol(value, wideFontName, mode, options,
+			["mord", wideFontClass]);
```
Comment (ylemkimon) [inline src/buildCommon.js]: How about `classes.concat(wideFontClass)`?

Comment (ronkok) [reply src/buildCommon.js]: @ylemkimon Yes, that will work. Is that faster?

Comment (kevinbarabash) [reply src/buildCommon.js]: The benefit is that if someone adds something to `classes` they don't have to also update this line.

Comment (ronkok) [reply src/buildCommon.js]: Ah, that makes sense. Thank you for pointing it out.

**Pull request merged**